.gitpod.yml

```diff
@@ -1,23 +1,19 @@
 ---
-image: gitpod/workspace-base
+image:
+    file: .gitpod.Dockerfile
+
 # Commands that will run on workspace start 
 tasks: 
-    - name: Setup, Install & Build 
-      before: apt install cargo redis-server nodejs npm -y && cargo test
-      init: cargo install cargo-watch 
-      command: redis-server --port 8080 & cargo watch -q -w "." -x "run" 
-# Ports to expose on workspace startup 
-ports: 
-    - name: Website 
-      description: Website Preview 
-      port: 8080 
-      onOpen: open-preview
```

Comment:
Also, I think this should be added back again in the config as this will automatically launch the browser which allows testing the changes quickly and easily. What do you say ?

Reply:
It doesn't work I already tested.

Reply:
Ok, I see, but if it can be added then it would be great. For this, I would like to suggest this documentation page on how to set up previewing in Gitpod:

https://www.gitpod.io/docs/configure/workspaces/ports

Reply:
There are some issues with preview in Gitpod. And we do get a notification when the port is open and have the option to either open-preview or open-browser.

Reply:
Ok, but I had the intent that if it were to be automatic that would have been better. I think we can figure out this part in the future. :slightly_smiling_face:

Reply:
Yes, I will talk about it in their discord. Let's see.